app/src/main/java/at/ac/fhcampuswien/budget_fox/widgets/SimpleDropdownField.kt

```diff
@@ -0,0 +1,58 @@
+package at.ac.fhcampuswien.budget_fox.widgets
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun SimpleDropdownField(
+    placeholder: String,
+    items: List<String>,
+    onItemSelected: (String) -> Unit
+) {
+    var expanded by remember { mutableStateOf(false) }
+    var selectedItem by remember { mutableStateOf<String?>(null) }
+
+    Column {
+        Box(modifier = Modifier.fillMaxWidth().padding(top = 10.dp)) {
+            BasicText(
+                text = selectedItem ?: placeholder,
+                style = MaterialTheme.typography.bodyLarge.copy(color = Color.Black),
+                modifier = Modifier
+                    .clickable { expanded = true }
+                    .padding(20.dp)
+                    .fillMaxWidth()
+            )
+            DropdownMenu(
+                expanded = expanded,
+                onDismissRequest = { expanded = false }
+            ) {
+                items.forEach { label ->
+                    DropdownMenuItem(
+                        { BasicText(text = label) },
```

Comment:
```suggestion
                        { Text(text = label, color = MaterialTheme.colorScheme.inverseSurface) },
```
<img width="136" alt="Bildschirmfoto 2024-06-26 um 13 46 55" src="https://github.com/EP1210/Budget_Fox/assets/58122000/e28b68df-0fb6-4c1d-ba0b-4e8741800770">